CHANGES

```diff
@@ -170,6 +172,9 @@ skyview
 solarsystem
 ^^^^^^^^^^^
 
+- updates to jplhorizons [#1335, #1478] and mpc [#1350]
```

Comment:
I'm not sure how to deal with the changelog for the aggregating modules. shall we just move the entries from e.g. ``jplhorizon`` to ``solarsystem.jplhorizon``? I'm not sure. @keflavich?

Reply:
sure, why not?  Or just keep them all under `solarsystem` and say stuff like:
```
solarsystem
^^^^^^^^^^^^
- jplhorizons: updates to blahblah [#1335]
- mpc: updates to foobar [#1350]
```
since it is unlikely these subsections in the changelog will grow out of hand

Reply:
Another way would be to simply remove `solarsystem` here, as it simply serves as a collection of tools in other models. I guess this depends on how you want to organize the modules in the future...

Reply:
in astropy we have `astropy.io.ascii`, `astropy.io.fits`, etc. listed in alphabetical. For some of the modules here that will make sense, e.g. for `esa.gaia` or `cds.vizier`, but I suppose it doesn't make sense for `solarsystem.mpc` or `observatory_archives.cadc`.

While probably it worth having a `solarsystem` entry when a new module is added to that collection. 

Does it make sense?

Reply:
Ok, so changes to `jplhorizons` and `mpc` go under the corresponding module header and in a short form under `solarsystem`? Or do we use only one of the aforementioned?

Reply:
no, I think the former is enough, and `solarsystem` only gets an entry when e.g. `imcce` is added. No need to change here anything, I'll clean up the changelog before tagging the release.

Reply:
got it, thanks!